cmd/relay/main.go

```diff
@@ -12,12 +12,21 @@ import (
 
 func main() {
 	socksAddr := flag.String("socks", "localhost:8000", "Address and port of SOCKS5 proxy server.")
+	socksUsername := flag.String("username", "", "Username of SOCKS5 proxy server.")
```

Comment:
Change the flag name to '-socks-user', such that it is clear that it is the username for access to the socks proxy.

Reply:
Agree with your opinion, it has been changed.